bentoml/yatai/client/deployment_api.py

```diff
@@ -366,6 +368,80 @@ def list_sagemaker_deployments(
         list_result.deployments.extend(sagemaker_deployments)
         return list_result
 
+    def create_ec2_deployment(
+        self,
+        name,
+        bento_name,
+        bento_version,
+        region,
+        min_capacity,
+        desired_capacity,
+        max_capacity,
+        instance_type,
+        ami_id,
```

Comment:
Should all of these values to be required from the user? I think we can provide a good default value for few of them such as min_capacity, max_capacity, instance_type, desired_capacity to help users have an easier time to use it.

You can find an example with auzre function: https://github.com/bentoml/BentoML/blob/c4241435bbfe5eb4e12476e7d8f15e1683ba6ec4/bentoml/yatai/client/deployment_api.py#L473

Reply:
I didn't get this.
If values are not provided on cli,defaults are passed to this.
Do you still want to make these input as keyword instead of positional?

Reply:
Sorry, I missed this message.

This client API could be called from user's python function as well.  Let's make them optional keyword arguments and set the missing value to defaults.